### ISSUE TITLE
Nice

### ISSUE THREAD

**Issue by Erikyy** [AndreasOun/TODOlist#1]
this is easy, you should be able to complete this very quickly,

**Comment by AndreasOun** [comment]
> this is easy, you should be able to complete this very quickly, next try this https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch and display data as a list on html

Yeah, trying to do at least 5 javascript vanilla projects before moving into React. 
Thanks for the link!
